Haulage/DatabaseSetup.cs

```diff
@@ -0,0 +1,62 @@
+using System.Data.SQLite;
+using System.IO;
+
+public static class DatabaseSetup
+{
+    private static string GetDatabasePath()
+    {
+        // Get the base directory of the application
+        string baseDirectory = AppDomain.CurrentDomain.BaseDirectory;
+
+        // Combine the base directory with the relative path to the database
+        string relativePath = Path.Combine(baseDirectory, @"..\..\..\..\..\Database\Haulage.db");
+
+        // Resolve to a full absolute path
+        return Path.GetFullPath(relativePath);
+    }
+
+    private static string connectionString = $"Data Source={GetDatabasePath()};Version=3;";
+
+
+    public static void InitializeDatabase()
+    {
+        if (!File.Exists($@"{GetDatabasePath()}"))
+        {
+            SQLiteConnection.CreateFile($@"{GetDatabasePath()}");
+
+            using (var connection = new SQLiteConnection(connectionString))
+            {
+                connection.Open();
+
+                // Create User Table for Haulage Data
+                string createUsersTableQuery = @"
```

Comment:
this should be put in a list<string> called commands and the commands should be looped through

Reply:
Agreed for multiple tables we should be looping through commands, will update when adding additonal tables.